CONTRIBUTING.md

```diff
@@ -0,0 +1,276 @@
+   execute `git pull --rebase` under the hood. Please do your part to keep our commit history clean and meaningful!
+2. The flip side of #1: **do not *ever* try to rebase or amend commits that are already on a remote branch**. No force
+   pushing, _ever_, unless you know _exactly_ what you are doing (if you know what you're doing, you probably don't need
+   to force push!).
+3. **Prepend Feature and Bugfix branches with the relevant issue ID**. For example, `feature/600-new-cool-thing` or 
+   `bugfix/601-horrible-terrible-bug`. It's a small thing, but it helps us better understand what you're working on.
+4. **Use Pull Requests after publishing feature/bugfix branches.** We use PRs for code review. Since our `origin` uses
+   the default strategy of `--no-ff` merges ([configured][merge-methods-github] in GitHub), you can press the merge
+   button or issue a local `git flow ... finish` without *ever* having to force push. 
+
+#### Forking
+While core contributors will work directly on the original repo, outside contributors will need to maintain their own 
+fork of Lookit while working on new features for (or fixing bugs for) the platform. We recommend reading this handy
+[guide on forking][forking-workflow-walkthrough] if you are not familiar with this workflow. GitHub's own [`hub` 
+utility][hub-oss-contrib] will make your life a lot easier in this regard by abstracting over some of the steps; you 
+can find the installation instructions [here][hub-install] (or, if you like to live on the cutting-edge, try out 
+GitHub's [shiny new CLI][github-cli]).
+
+#### Staying on the Command Line
+If you have `hub` and `gitflow-avh` installed, you should be able to work entirely from the command line - even when
+working with a PR of a published feature/bugfix branch.
+
+You'll need to change a bit of `gitflow-avh`'s default `finish` behavior to push to remote branches:
+```bash
+git config --local gitflow.bugfix.finish.push yes
+git config --local gitflow.feature.finish.push yes
+git config --local gitflow.release.finish.push yes
+```
+This will enact proper Git Flow "finalizer" processes on remote branches (e.g. merging `release` branches back to 
+`develop` and `master`, deleting old `feature` and `bugfix` branches). This way, you won't have to do manual (remote) 
+branch cleanup with vanilla git commands, or have to remember to push tags.
+
+Once that's done, you'll just do normal Git Flow process with one exception: after running a `git flow ... publish` 
+command, you'll run `hub pull-request` and fill out a proper PR with your command line editor of choice.
+
+Since GitHub considers a PR "Merged" as soon as the commits in the PR branch are also found in the target branch (so
+long as the commits are not modified in any way - not a problem if you never rebase or amend remote commits!), you can 
+just issue a `git flow ... finish` command to do the necessary merges, tag pushes, and branch cleanup.
```

Comment:
this is aimed only at core contributors, right? (or at least parts - outside contributors shouldn't be trying to merge release branches back into develop or master...) If so can we denote that more clearly?

Reply:
Changing the headings to clearly reflect intended audiences